app/src/main/java/ru/netology/fmhandroid/adapter/ClaimCommentListAdapter.kt

```diff
@@ -0,0 +1,106 @@
+package ru.netology.fmhandroid.adapter
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.DiffUtil
+import androidx.recyclerview.widget.ListAdapter
+import androidx.recyclerview.widget.RecyclerView
+import ru.netology.fmhandroid.R
+import ru.netology.fmhandroid.databinding.ItemCommentBinding
+import ru.netology.fmhandroid.dto.ClaimCommentWithCreator
+import ru.netology.fmhandroid.dto.User
+import ru.netology.fmhandroid.utils.Utils
+
+
+interface OnClaimCommentItemClickListener {
+    fun onCard(claimComment: ClaimCommentWithCreator) {}
+}
+
+// Временная переменная. После авторизации заменить на залогиненного юзера
+val user = User(
+    id = 1,
+    login = "User-1",
+    password = "abcd",
+    firstName = "Дмитрий",
+    lastName = "Винокуров",
+    middleName = "Владимирович",
+    phoneNumber = "+79109008765",
+    email = "Vinokurov@mail.ru",
+    deleted = false
+)
+
+class ClaimCommentListAdapter(
+    private val onClaimCommentItemClickListener: OnClaimCommentItemClickListener
+) :
+    ListAdapter<ClaimCommentWithCreator, ClaimCommentListAdapter.ClaimCommentViewHolder>(
+        ClaimCommentDiffCallback()
+    ) {
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): ClaimCommentViewHolder {
+        val binding = ItemCommentBinding.inflate(
+            LayoutInflater.from(parent.context),
+            parent,
+            false
+        )
+        return ClaimCommentViewHolder(binding, onClaimCommentItemClickListener)
+    }
+
+    override fun onBindViewHolder(holder: ClaimCommentViewHolder, position: Int) {
+        getItem(position)?.let {
+            holder.bind(it)
+        }
+    }
+
+    class ClaimCommentViewHolder(
+        private val binding: ItemCommentBinding,
+        private val onClaimCommentItemClickListener: OnClaimCommentItemClickListener
+    ) : RecyclerView.ViewHolder(binding.root) {
+
+        fun bind(claimComment: ClaimCommentWithCreator) {
+            binding.apply {

```

Comment:
На случай, если захочется поэкономить место и не вызвать побочных эффектов
```suggestion
        fun bind(claimComment: ClaimCommentWithCreator) = with(binding) {
```